tests/h/services/group_test.py

```diff
@@ -5,344 +5,408 @@
 import mock
 import pytest
 
-from h.models import Group, GroupScope
+from h.models import Group, User, GroupScope
 from h.models.group import JoinableBy, ReadableBy, WriteableBy
 from h.services.group import GroupService
 from h.services.group import groups_factory
-from h.services.organization import OrganizationService
-
+from h.services.user import UserService
 from tests.common.matchers import Matcher
 
 
-class TestGroupService(object):
-    def test_create_private_group_returns_group(self, service):
-        group = service.create_private_group('Anteater fans', 'cazimir')
+class TestGroupServiceCreatePrivateGroup(object):
```

Comment:
It might be worth a docstring:

    """Unit tests for :py:meth:`GroupService.create_private_group`"""

just to clarify the class name.